ramp-utils/ramputils/password.py

```diff
@@ -0,0 +1,38 @@
+import bcrypt
+
+from .string_encoding import encode_string
+
+
+def hash_password(password):
```

Comment:
this is only used by rampdb package, and not by any else? (or do we plan to use it somewhere else?) 
Otherwise I would move it to rampdb.utils

Reply:
It is used in the front-end as well but I agree with your point to limit the number of packages.

Reply:
I can move it in utils thought